core/platform-api/che-core-api-machine/src/main/java/org/eclipse/che/api/machine/shared/Constants.java

```diff
@@ -36,8 +36,10 @@
     public static final String LINK_REL_GET_MACHINE_LOGS_CHANNEL   = "get machine logs channel";
     public static final String LINK_REL_GET_MACHINE_STATUS_CHANNEL = "get machine status channel";
 
-    public static final String WS_AGENT_SERVER_REFERENCE = "extensions";
+    public static final String WS_AGENT_REFERENCE                  = "workspace-agent";
```

Comment:
We already used "wsagent" instead of "workspace-agent" [here](https://github.com/eclipse/che/pull/635/files#diff-7f9617a5d539c8852159d21362524d3fR44). I believe we should use one term in links.
@gazarenkov WDYT?